doc/ale-glsl.txt

```diff
@@ -0,0 +1,36 @@
+===============================================================================
+ALE GLSL Integration                                         *ale-glsl-options*
+                                                         *ale-integration-glsl*
+
+===============================================================================
+Integration Information
+
+  Since Vim does not detect the glsl file types out-of-the-box, you need the
+  runtime files for glsl from here: https://github.com/tikhomirov/vim-glsl
+
+  Note that the current glslang-based linter expects glslangValidator in
+  standard paths. If it's not installed system-wide you can set
+  |g:ale_glsl_glslang_executable| to a specific path.
+
+
+===============================================================================
+glslangValidator                                    *ale-glsl-glslangValidator*
```

Comment:
I recommend changing these to `glslang` and `*ale-glsl-glslang*`. See my other comment.